### DISCUSSION TITLE
how to fix

### DISCUSSION THREAD

**Discussion by eliekh05** [utmapp/UTM#4751] [UTM Help (iOS)] [answered]
hi i am using A1863 iOS 16.1.2 NOT JAILBROKEN and i installed latest version UTM.ipa and i want to open the app it will give me this error how can i fix it

 
![5A317C8A-C932-4AA8-BB88-2D8BA7229739](https://user-images.githubusercontent.com/67870780/205447064-b1f00fbc-abca-4ad0-81aa-69fbed58d1f4.png)

**Comment by osy** [comment, marked as answer]
Follow the link.

**Reply by eliekh05** [reply]
it is just how to install it

**Reply by osy** [reply]
Read the whole page and you’ll have your answer.
